Symphony/Composer.swift

```diff
@@ -30,8 +30,18 @@ public protocol Composer: class {
 public extension Composer {
 
     public func present(_ composable: Composable, animated: Bool = false) {
+
         currentComposables.append(composable)
-        containerViewController.present(composable.viewController, animated: animated, completion: nil)
+
+        let presentingViewController: UIViewController
+
+        if let presentedViewController = containerViewController.presentedViewController {
```

Comment:
Minor style quibble, but this can be tackled pretty cleanly in a single expression:
`let presentingViewController = containerViewController.presentedViewController ?? containerViewController`

Reply:
Ah, awesome catch.  I had the thought that there had to be a shorter hand for that, but wasn't seeing that it could be done with a simple `??`